sdk/tests/deqp/functional/gles3/es3fLifetimeTests.js

```diff
@@ -100,8 +100,12 @@ es3fLifetimeTests.ScaleProgram.prototype.draw = function(vao, scale, tf, dst) {
 es3fLifetimeTests.ScaleProgram.prototype.setPos = function(buffer, vao) {
     gl.bindBuffer(gl.ARRAY_BUFFER, buffer);
     gl.bindVertexArray(vao);
-    if (buffer)
+    if (buffer) {
         gl.vertexAttribPointer(this.m_posLoc, NUM_COMPONENTS, gl.FLOAT, false, 0, 0);
```

Comment:
We should probably change WebGL 2 spec, and when a vao is bound, and nothing is bound to ARRAY_BUFFER, let the vertexAttribPointer(offset == 0) to be accepted.

Let's merge this for now.